<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>21</y>
      <width>561</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="playBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>⏵</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>480</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>⏸</string>
    </property>
   </widget>
   <widget class="QPushButton" name="forwardBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>480</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>⏩</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>⏪</string>
    </property>
   </widget>
   <widget class="QLabel" name="coverView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>450</y>
      <width>741</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
